guides/user/plugins/user-guide.md

```diff
@@ -7,103 +7,77 @@ redirect_from:
   - /guides/user/plugin-users/
 ---
 
-{% include alpha version="1.19.4" %}
+{% include alpha version="1.20.6" %}
 
->Note: Spinnaker 1.19.x only supports backend plugins due to a bug in Deck.
+_Note: Spinnaker 1.19.x does not support frontend plugins due to a bug in Deck._
 
 {% include toc %}
 
-In this guide, you add an existing plugin from an [example repository](https://github.com/spinnaker-plugin-examples/examplePluginRepository) to Spinnaker. See the [Plugin Creators Guide](/guides/developer/plugin-creators/overview/) for how to create a plugin.
+## Overview
 
-# Requirements
+Spinnaker uses [PF4J-Update](https://github.com/pf4j/pf4j-update) to load and manage third-party plugins. These plugins can implement a PF4J extension point or be Spring components. See the [Plugin Creators Guide](/guides/developer/plugin-creators/overview/) for details.
+
+## Terms
+
+**plugins.json**
+
+* required file
+* defines one to many plugins in a plugin repository
+* each plugin definition has an id, description, provider, and a collection of releases (version, date, requires, sha512sum, state, url)
+* the plugin developer provides access to this file
+
+**repositories.json**
+
+* optional file
+* defines one to many plugin repositories
+* each repository definition consists of a unique identifier and the path to a `plugins.json` file
+* the plugin developer may supply this file
+
+## Plugin requirements
 
 * The plugin is either a [Plugin Framework for Java](https://github.com/pf4j/pf4j)(PF4J) plugin or a Spring plugin
-* The plugin resides in a location that Spinnaker can access
-* You use Spinnaker v1.19.4 or later
-* You use Halyard v1.34 or later to deploy Spinnaker
+* The plugin repository is a web location that Spinnaker can access, like a GitHub repository
 
-# Plugins overview
+Spinnaker environment:
 
-Spinnaker uses [PF4J-Update v2.3.0](https://github.com/pf4j/pf4j-update) to load and manage third-party plugins. Spinnaker supports local, remote, and file system repositories.
+* Spinnaker v1.20.6 or later
+* Halyard v1.36 or later to deploy Spinnaker
 
-Each plugin repository is defined in a `repositories.json` file. Spinnaker must have access to a repository's location. A repository contains a list of one or more plugins defined in a `plugins.json` file. Spinnaker reads the `repositories.json` file and then loads enabled plugins from the corresponding `plugins.json` file.
 
-See the PF4J-Update [README](https://github.com/pf4j/pf4j-update/tree/release-2.3.0) for a full explanation of the `repositories.json` and `plugins.json` structures.
+## How to add a plugin to Spinnaker
 
-# Define plugins
+1. Add a plugin repository using Halyard
```

Comment:
Looks like you're repeating these as headings, below. Maybe link each of these steps to the corresponding heading?